serial.ts

```diff
@@ -263,7 +263,7 @@ export class SerialPort {
                 this.writable_ = null;
               }),
           new ByteLengthQueuingStrategy({
-            highWaterMark: this.serialOptions_.bufferSize ?? kDefaultBufferSize,
+            highWaterMark: this.serialOptions_.bufferSize ?? this.outEndpoint_.packetSize > kDefaultBufferSize ? this.outEndpoint_.packetSize : kDefaultBufferSize,
```

Comment:
This change is unnecessary because babble errors don't apply to OUT transfers.

Reply:
ok